charts/cp-schema-registry/values.yaml

```
@@ -162,7 +161,8 @@ prometheus:
 
 ui:
   annotations: {}
-  labels: {}
+  labels:
+    sidecar.istio.io/inject: "false"
```

Why disable istio on the UI ?
Before this PR, it was not disabled and (AFAIK) the UI works fine. So I suggest to keep it enabled.

Thus this suggest to just rollback this change:
```suggestion
  labels: {}
```